libindy/src/errors/wallet.rs

```diff
@@ -18,12 +23,20 @@ pub enum WalletError {
     AlreadyExists(String),
     NotFound(String),
     IncorrectPool(String),
-    PluggedWallerError(ErrorCode),
+    PluggedWalletError(ErrorCode),
```

Comment:
I don't understand what is the difference between PluggedWalletError and StorageError

Reply:
PluggedWalletError was left for now, until we finish plugged wallet and figure out whether we should remove it. 
By that time, it will probably be removed